test/testcases/lineends/issue_231_anon_function_parameter.hxtest

```diff
@@ -24,14 +24,14 @@ class Main  {
 
 class Main {
 	function main() {
-		owner.addEntity(new Entity()
-			.addComponent(_sprite = FlumpAssets.getMovieSprite("mute_button", LibraryName.INTERFACE)).addComponent(new PointerTap(function(e) {
-			trace("yes");
-		})));
-		owner.addEntity(new Entity()
-			.addComponent(_sprite = FlumpAssets.getMovieSprite("mute_button", LibraryName.INTERFACE)).addComponent(new PointerTap(function(e:Any) {
-			trace("yes");
-		})));
+		owner.addEntity(new Entity().addComponent(_sprite = FlumpAssets.getMovieSprite("mute_button", LibraryName.INTERFACE))
+			.addComponent(new PointerTap(function(e) {
+				trace("yes");
+			})));
+		owner.addEntity(new Entity().addComponent(_sprite = FlumpAssets.getMovieSprite("mute_button", LibraryName.INTERFACE))
+			.addComponent(new PointerTap(function(e:Any) {
+				trace("yes");
+			})));
```

Comment:
Regression?

Reply:
no, change of default option from `fillLine` to `onePerLineAfterFirst`, because all other options also use `onePerLineAfterFirst` so it's more consistent

Reply:
I mean the indentation.